host/src/FunctionsNetHost/Prelaunch/Prelauncher.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Diagnostics;
+
+namespace FunctionsNetHost.Prelaunch
+{
+    internal static class Prelauncher
```

Comment:
Open to better names for the type and method.

Reply:
just capitalize the 'L' maybe? otherwise I'm fine with it...

Reply:
Renamed to PreLauncher.